pom.xml

```diff
@@ -52,8 +52,8 @@
     </dependencies>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
Is it possible to choose JDK 11 for this? If we update this to 17 it will no longer be compatible with JDK 11?
Is it possible to keep this at 1.8 or maybe 9? As an open source project (even though we're maintaining this clone of it) maybe we can choose a JDK that's more widely adopted?
Not sure if this will be an issue, but would we potentially need to fix bugs here that need to be used in earlier versions of Engine?

Reply:
I guess we need to have a branch maybe just called jnode-fs-jdk17 for this change, so we can maintain both of them separately. Or a tag?

I saw aspose do it like below, I don't know how they separate the code base.
val aspose_imaging get() = dependencies.create("com.aspose:aspose-imaging:19.10:jdk16")
val aspose_pdf get() = dependencies.create("com.aspose:aspose-pdf:20.7:jdk17")

Yeah, we probably need another branch for jdk17.